board_similarity.py

```diff
@@ -0,0 +1,161 @@
+import chess
+import chess.engine
+import os
+import math
+import random
+from time import time
+
+
+def score_similarity(score1, score2, same_players=True, similarity_range=2000):
+    """
+    :param score1: score of first fen, in lan format, convert to str
+    :param score1: score of second fen, in lan format, convert to str
+    :param same_players: says whether in both cases,
+    the move is for the player of the same color
+    :param similarity_range: hyperparameter, int
+    :return: similarity of score for given items in the range from -1 to 1
+    """
+    if score1[0] == "#" or score2[0] == "#":
+        if score1[0] == "#" and score2[0] == "#":

```

Comment:
duplicate condition?

Reply:
There are "or" and "and" in this condicions. can it be distinguished in some other way?

Reply:
sorry, I didn't notice